src/EnergyPlus/HVACHXAssistedCoolingCoil.cc

```diff
@@ -1895,9 +1656,6 @@ namespace HVACHXAssistedCoolingCoil {
         // incorrect coil type or name is given, ErrorsFound is returned as true and capacity is returned
         // as negative.
 
-        // Using/Aliasing
-        auto &GetWaterCoilMaxFlowRate(WaterCoils::GetCoilMaxWaterFlowRate);
```

Comment:
[src/EnergyPlus/HVACHXAssistedCoolingCoil.cc:1899]:(style),[constVariable],Variable 'GetWaterCoilMaxFlowRate' can be declared as reference to const